crates/codegen/schema/src/types/production.rs

```diff
@@ -82,7 +82,7 @@ impl Production {
 #[serde(deny_unknown_fields, rename_all = "camelCase")]
```

Comment:
Are we now able to remove the `TODO` comment/hack in `get_exact_version()`? After this PR, it will return the wrong versions.

Reply:
done